builders.ncl

```diff
@@ -1,5 +1,12 @@
-let { NickelDerivation, Derivation, .. } = import "contracts.ncl" in
-
+let { NickelDerivation, Derivation, NixString, .. } = import "contracts.ncl" in
+
+let concatStringsSep = fun sep values =>
```

Comment:
In general we tend to use snake_case for identifiers, and CamelCase for types/contracts.

```suggestion
let concat_strings_sep = fun sep values =>
```